test/models/real-fevr-marketplace.spec.ts

```diff
@@ -0,0 +1,78 @@
+import {defaultWeb3Connection, erc20Deployer, revertChain} from '../utils/';
+import {expect} from 'chai';
+import {RealFevrOpener} from '@models/real-fevr-opener';
+import {RealFevrMarketplace} from '@models/real-fevr-marketplace';
+import {toSmartContractDecimals} from '@utils/numbers';
+
+describe('Marketplace RealFevr', async () => {
+  let marketplaceContract: RealFevrMarketplace;
+  let tokenAddress!: string;
+  let openerAddress!: string;
+  let contractAddress!: string;
+  let accountAddress!: string;
+
+  const purchaseTokenCap = toSmartContractDecimals(1000000) as number;
+  const web3Connection = defaultWeb3Connection();
+
+  before(async () => {
+    await web3Connection.start();
+    await revertChain(web3Connection.Web3);
+    accountAddress = web3Connection.Account.address;
+  });
+
+  it(`Deploys the contract`, async () => {
+    const erc20 = await erc20Deployer(`Settler`, `$settler`, purchaseTokenCap, web3Connection);
+    tokenAddress = erc20.contractAddress!;
+
+    const openerDeployer = new RealFevrOpener(web3Connection);
+    openerDeployer.loadAbi();
+    const openerTx = await openerDeployer.deployJsonAbi(`Collectible`, `$stamp`, tokenAddress);
+
+    expect(openerTx.blockNumber).to.exist;
+    openerAddress = openerTx.contractAddress!;
+
+    const marketplaceDeployer = new RealFevrMarketplace(web3Connection);
+    marketplaceDeployer.loadAbi();
+    const marketplaceTx = await marketplaceDeployer.deployJsonAbi(openerAddress, tokenAddress);
+
+    expect(marketplaceTx.blockNumber).to.exist;
+    contractAddress = marketplaceTx.contractAddress;
+  });
+
+  describe('Methods', () => {
+    before(async () => {
+      marketplaceContract = new RealFevrMarketplace(web3Connection, contractAddress!, openerAddress, tokenAddress);
+      await marketplaceContract.loadContract();
+    });
+
+    it('should create pack of NFTs', async () => {
+      const res = await marketplaceContract.opener.createPack(1, 1, '', 'foo', 'bar', 1, [ '0x0000000000000000000000000000000000000000' ], [ 100 ], [], []);
+      expect(res).to.not.equal(false);
```

Comment:
`res` is probably a `TransactionReceipt` so it will _never_ be false. `res` will only have a primitive value if the method is a _pure call function_ (functions that change state CAN'T return a value -- they all return a TxReceipt) 

You can use the `hasTxBlockNumber` function exported from `test/utils/`

Reply:
done, thanks for the tip 👍